### ISSUE TITLE
please provide a post example

### ISSUE THREAD

**Issue by ksipma** [ccvshop/python-example-code#1]
Please provide a POST example. if i dump a json object of my request as data in the headers it doesn't validate... 401.20

**Comment by barbaart** [comment]
i agree with @ksipma. Would be great if you provide a POST or PATCH request example in PHP/PYTHON. I just copied the "post a product" example from "https://demo.ccvshop.nl/API/Docs/" to post to our store, but i'm also getting a 401.20..
